src/wagtail_ai/static_src/utils.tsx

```diff
@@ -5,11 +5,27 @@ import {
   ContentState,
   RichUtils,
 } from 'draft-js';
-
-import type { Prompt } from './custom';
+import { camelizeKeys } from 'humps';
+import type { Prompt, WagtailAiConfiguration } from './custom';
 
 class APIRequestError extends Error {}
 
+export const getAIConfiguration = (): WagtailAiConfiguration => {
+  const configurationElement =
+    document.querySelector<HTMLScriptElement>('#wagtail-ai-config');
+  if (!configurationElement || !configurationElement.textContent) {
+    throw Error('No wagtail-ai configuration found.');
+  }
+
+  try {
+    return camelizeKeys(
```

Comment:
**Suggestion:** The addition of a dependency just for this doesn't seem worth it. If we want camel case in JS, we could provide an already camel-case'd dict from Python?

Reply:
Refactored in 73506050e4cd6b27dc7b541ed96f0c5747425842 and removed dependency.